inst/qml/RobustBayesianMetaAnalysis.qml

```diff
@@ -75,61 +75,61 @@ Form
 		{
 			label:		qsTr("Correlations & N")
 			value:		"correlation"
-			id:			measures_correlation
+			id:			measuresCorrelation
 			enabled:	mainWindow.dataAvailable
 		}
 
 		RadioButton
 		{
 			label:		qsTr("Odds ratios & CI")
 			value:		"OR"
-			id:			measures_OR
+			id:			measuresOR
 			enabled:	mainWindow.dataAvailable
 		}
 
 		RadioButton
 		{
 			label:		qsTr("Effect sizes & SE")
 			value:		"general"
-			id:			measures_general
+			id:			measuresGeneral
```

Comment:
Don't you want to call this measuresEffectSizesAndSE or measuresEffectSizes? General is not that descriptive

Reply:
the Cohen's d measured effect sizes can be also specified using ES and SE, this prevents confusion since each of them is evaluated differently.

Reply:
the alternative is having 5 analysis, each of them dedicated to different input options which is much worse in my opinion

Reply:
I think that having 5 separate analyses is much better, because it would be more in style with the other analyses. Furthermore, the user then doesn't have to bother with redundancies (no standard errors needed for correlation for example) making the UI much cleaner and more intuitive. 

The analogue of doing everything in one analysis is to remove t-test, anova, ancova, linear regression, glm,  glmm, and only offering sem.

Reply:
> the user then doesn't have to bother with redundancies 

that's not true, the analysis run when the necessary input is specified - it does not require the redundant output to be filled

> The analogue of doing everything in one analysis is to remove t-test, anova, ancova, linear regression, glm, glmm, and only offering sem.

In my opinion, it is equal to splitting GLMM into GLMM binomial, GLMM poison, GLMM gamma, GLMM normal, GLMM beta....